### ISSUE TITLE
There are horizontal stripes in my result images.

### ISSUE THREAD

**Issue by lucid0921** [junyanz/pytorch-CycleGAN-and-pix2pix#860]
My scripts:

python train.py --dataroot datasets/AU2TC/ --name AU2TC_cyclegan --model cycle_gan --phase train  --batch_size 28 --preprocess none --load_size 256 --print_freq 1000 --save_epoch_freq 50

python test.py --dataroot datasets/AU2TC/ --name AU2TC_cyclegan --model cycle_gan --phase test  --batch_size 28 --preprocess none --load_size 256 --num_test 9056

After I ran the test, I got results made of image sets (real_A, fake_B, rec_A, real_B, fake_A, rec_B).

Images of names "*fake_B*" had undesirable results. They have horizontal stripes, even in the same position!

There were no white lines in fake_A images. 

How can I fix it. My fake_B images are very important to me. Please help me.

![image](https://user-images.githubusercontent.com/34934336/69813221-169a5080-1235-11ea-86cb-b6ed4653e295.png)

**Comment by junyanz** [comment]
Not sure what has happened. Maybe reduce the learning rate?

**Comment by lucid0921** [comment]
Reducing the learning rate was not helpful, but large training sample size solved the strip problems.

**Comment by coded5282** [comment]
@lucid0921 Hello. I'm also getting white horizontal stripes in my generated images. May I ask if you solved the issue by increasing the batch size? Or was it with more training data?

**Comment by lucid0921** [comment]
According to my previous mention, larger training set seems to have helped in solving the issue of the white horizontal line at that time. But honestly, at that time, I was immersed to go to the next step in my analysis so it is not certain whether the larger training set was key among many tries I did. Let me know your solution later. Thank you.

**Comment by coded5282** [comment]
Thank you. I've also recently gone to my next stage, but increasing the training set did seem to help a bit. I seemed to be getting a lot more horizontal lines in the later epochs, so increasing the training set seemed to delay that even further.
